sp800-63b/sec5_authenticators.md

```diff
@@ -344,39 +344,44 @@ Therefore, the use of biometrics for authentication is supported with the follow
 
 Biometrics SHALL be used with another authentication factor (something you have).
 
-Empirical testing of the biometric system to be deployed SHALL demonstrate an equal error rate of **1 in 1000** or better with respect to matching performance. The biometric system SHALL operate with a false match rate of **1 in 1000** or better.
+An authenticated protected channel between sensor (or endpoint containing a sensor that resists sensor replacement) and verifier SHALL be established and the sensor or endpoint authenticated **prior** to capturing the biometric sample from the claimant.
 
-When the biometric sensor and subsequent processing are not part of an integral unit that resists replacement of the sensor, the sensor (or endpoint with which it is an integral unit that resists sensor replacement) SHALL demonstrate that it is a certified or qualified sensor meeting these requirements by authenticating itself to the processing element.
+Empirical testing of the biometric system to be deployed SHALL demonstrate an equal error rate of **1 in 1000** or better with respect to matching performance. The biometric system SHALL operate with a false match rate of **1 in 1000** or better.
 
 The biometric system SHOULD implement presentation attack protection (PAD). Testing of the biometric system to be deployed SHOULD demonstrate at least 90% resistance to presentation attacks for each relevant attack type (aka species), where resistance is defined as the number of thwarted presentation attacks divided by the number of trial presentation attacks.
 
 >Note: Presentation attack detection is being considered as a mandatory requirement in future editions of this guideline.
 
 The biometric system SHALL allow no more than 3 consecutive failed authentication attempts, or 10 consecutive failed attempts if presentation attack detection meeting the above requirements is implemented. Once that limit has been reached, the claimant SHALL be required to use a different authenticator or to activate their authenticator with a different factor such as a memorized secret.
 
+**At AAL2**, the verifier SHOULD make a determination that the biometric sensor and subsequent processing meet the performance requirements stated above. **At AAL3**, this determination SHALL be performed.
+
+For any AAL, determination of sensor/endpoint performance, integrity, and authenticity can be accomplished in several different ways, any of which are acceptable under this guideline.  These include but are not limited to: authentication of the sensor or endpoint, testing and certification, and runtime interrogation of signed metadata (attestation) as described in Section [5.2.4](#attestation).
```

Comment:
how about 'certification by an accredited 3rd party'? This is better than what is there, and this is just an example.